### PR TITLE
creates css and html files in JuanSebastian folder

### PR DESCRIPTION
**Pull request by juancho11gm**
New files created in JuanSebastian folder. Topic: positioning

### CODE REVIEW

File: JuanSebastian/positioning.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.container {
+    margin-top: 50px;
+}
+
+.green-section {
+    border: 1px green;
+    border: solid;
+    border-color: green;
```
Comment (fabianmarinog) [inline JuanSebastian/positioning.css]: This can be simplified to: `border: solid 1px green;`

File: JuanSebastian/positioning.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    padding-bottom: 12px;
+    padding-top: 12px;
+    position: relative;
+    margin-right: 10px;
+}
+
+.nav-container {
+    border: 1px;
+    border: solid;
+    border-color: red;
```
Comment (fabianmarinog) [inline JuanSebastian/positioning.css]: This can be simplified to: `border: solid 1px red;`

File: JuanSebastian/positioning.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    margin-bottom: 1000px;
+    position: absolute;
+    width: 180px;
+}
+
+.top-header{
```
Comment (fabianmarinog) [inline JuanSebastian/positioning.css]: add space before {

File: JuanSebastian/positioning.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.top-header ul li {
+    display: inline-block;
+}
+
+.yellow-footer {
+    border: 1px;
+    border: solid;
+    border-color: yellow;
```
Comment (fabianmarinog) [inline JuanSebastian/positioning.css]: This can be simplified to: `border: solid 1px yellow;`

**Comment by juancho11gm** [comment]
I've done all of your suggestions.

**Pull request merged**